### PR TITLE
🐛 [TextField] Fix textarea unable to set value

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/TextField.vue [hunk 1]
```diff
@@ -2,13 +2,10 @@
   <div class="flex flex-col">
     <component
       :is="tag"
-      v-bind="$attrs"
-      :class="rootClasses"
-      :value="value"
```
Comment (AuroraHuang22) [inline components/TextField.vue:7]: I found that `:value.prop="value"` also works

Comment (nwingt) [reply components/TextField.vue:7]: You mean for textarea?

Comment (AuroraHuang22) [reply components/TextField.vue:7]: yes, I tried to only changed `:value="value"` to `:value.prop="value"` to solve the issue, and it works for me.

Comment (nwingt) [reply components/TextField.vue:7]: Still not working
https://user-images.githubusercontent.com/17264716/134216207-9948ebb7-9129-4070-a2be-8d435807e09b.mov

Comment (nwingt) [reply components/TextField.vue:7]: You may try setting a initial value for TextField rather than an empty string

**Review by AuroraHuang22 — APPROVED** [review]

**Pull request merged**